.circleci/config.yml

```diff
@@ -3,9 +3,10 @@ executors:
   small_executor:
     docker:
       - image: circleci/openjdk:11.0.8-jdk-buster
-        auth:
-          username: $DOCKER_USER_RO
-          password: $DOCKER_PASSWORD_RO
+        auth: &docker-auth
+          # Don't panic, throw away account to avoid Docker rate limits when downloading.
+          username: "cddockeruser"
+          password: "fa8651f2-88be-48b7-98ce-e711bd376252"
```

Comment:
is this an actual account we created? won't that mean someone can use those details for whatever they like? I mean even if it's throw-away, its still a real account...

Reply:
yes it's an actual account.  Just not one we care about.  People could indeed use it for other things but worst case Docker Hub revoke it and we have to create a new one.

There's no Teku assets or any association from that account to ConsenSys or Teku.  It's as valuable as any other account that anyone could create themselves.